Remove HTMLAttrxRef macro, part 1 (#25258)
mdn/content: files/en-us/web/html/element/img/index.md

## Image loading errors

> If an error occurs while loading or rendering an image, and an [`onerror`](/en-US/docs/Web/HTML/Global_attributes#onerror) event handler has been set on the {{domxref("Element/error_event", "error")}} event, that event handler will get called. This can happen in a number of situations, including:

Reviewer: ```suggestion
If an error occurs while loading or rendering an image, and an `onerror` event handler has been set for the {{domxref("Element/error_event", "error")}} event, that event handler will get called. This can happen in several situations, including:
```
